src/reducers/people.js

```diff
@@ -62,6 +63,14 @@ export default function (state = {}, action) {
         ...state,
         current: {...state.current, memberships}
       }
+    case UPDATE_COMMUNITY_SETTINGS_PENDING:
```

Comment:
nah, it's fine. we should be normalizing the data under currentUser so that memberships just has a list of ids, or break memberships itself into its own section of the store, but even then you would have to remove the membership.